CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * [#6865](https://github.com/rubocop-hq/rubocop/pull/6865): Fix deactivated `StyleGuideBaseURL` for `Layout/ClassStructure`. ([@aeroastro][])
 * [#6868](https://github.com/rubocop-hq/rubocop/pull/6868): Fix `Rails/LinkToBlank` auto-correct bug when using symbol for target. ([@r7kamura][])
 * [#6869](https://github.com/rubocop-hq/rubocop/pull/6869): Fix false positive for `Rails/LinkToBlank` when rel is a symbol value. ([@r7kamura][])
+* [#6785](https://github.com/rubocop-hq/rubocop/issues/6785): Do not register an offense for `Rails/Present` `UnlessBlank` and `Rails/Blank` `UnlessPresent` when `Style/UnlessElse` is enabled. ([@rrosenblum][])
```

Comment:
I think you messed up this sentence as I can't understand it. :-)

Reply:
Sorry about that. I had a hard time trying to convey the information in a concise way. I have updated the entry. Let me know what you think.